Sources/Networking/Caching/CustomerInfoCallback.swift

```diff
@@ -17,9 +17,9 @@ struct CustomerInfoCallback: CacheKeyProviding {
 
     typealias Completion = (Result<CustomerInfo, BackendError>) -> Void
 
-    let cacheKey: String
-    let source: NetworkOperation.Type
-    let completion: Completion
+    var cacheKey: String
+    var source: NetworkOperation.Type
+    var completion: Completion
```

Comment:
Could we add another init instead of making this mutable? Seems safer to me but lmk if you think there are problems with that.

Reply:
I'd recommend reading on Swift's value types. This isn't like having mutable data in other languages, it's only mutable for mutables copies, and even then, because it doesn't have reference semantics, you can't have spooky action at a distance by changing one "instance" and mutating something else. So it's perfectly safe.